.travis.yml

```
@@ -21,23 +20,36 @@ matrix:
     name: Linux Build using gcc & tests & code coverage
     script: $WORKSPACE/scripts/linux/build.sh && $WORKSPACE/scripts/linux/test.sh
   - language: android
-    name: Android Build
-    os : linux
     dist: xenial
-    language: cpp
+    jdk: oraclejdk8
+    name: Android Build
+    android:
+      components:
+      - build-tools-28.0.3
+      - android-28      
+      - extra-google-m2repository
+      - extra-android-m2repository # for design library
+      licenses:
+      - android-sdk-preview-license-.+
+      - android-sdk-license-.+
+      - google-gdk-license-.+    
+    install:
+    - echo y | sdkmanager 'cmake;3.6.4111459'
     before_script:
-    - wget https://dl.google.com/android/repository/android-ndk-r19c-linux-x86_64.zip
-    - wget https://dl.google.com/android/repository/android-21_r01.zip
-    - unzip -qq android-ndk-r19c-linux-x86_64.zip
-    - unzip -qq android-21_r01.zip
-    - export ANDROID_SDK_ROOT="$TRAVIS_BUILD_DIR/android-5.0"
-    - export ANDROID_NDK_HOME="$TRAVIS_BUILD_DIR/android-ndk-r19c"
+    - wget https://dl.google.com/android/repository/android-ndk-r20-linux-x86_64.zip
+    - wget https://dl.google.com/android/repository/platform-28_r06.zip
+    - unzip -qq android-ndk-r20-linux-x86_64.zip
+    - unzip -qq platform-28_r06.zip
+    - export SDK_ROOT="$TRAVIS_BUILD_DIR/android-9"
```

```suggestion
    - export ANDROID_HOME="$TRAVIS_BUILD_DIR/android-9"
```
I suggest to use `ANDROID_HOME` instead `SDK_ROOT`, cause it is more specific.